### PR TITLE
add docs for api-test

### PR DESCRIPTION
**Pull request by nabim777**
In this PR , docs is added which is about running API test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -574,6 +574,20 @@ cd $HOME/development/custom_apps/integration_openproject
...
 ```
 
+### API tests
+The API tests start a full Nextcloud server and openProject_integration apps.
+To run the tests locally
+1. Setup nextcloud and enable `integration_openproject` apps.
+2. Goto path `server/apps/integration_openproject`
+2. Run `NEXTCLOUD_BASE_URL=http://localhost/server make api-test`
+
+In order to run the specific scenrio only
```
Comment (phil-davis) [inline README.md:584]:
```suggestion
In order to run only a specific scenario
```

File: README.md [hunk 2]
```diff
@@ -574,6 +574,20 @@ cd $HOME/development/custom_apps/integration_openproject
...
+```shell
+NEXTCLOUD_BASE_URL=http://localhost/server \                                                                                                                                            ─╯
```
Comment (phil-davis) [inline README.md:586]:
```suggestion
NEXTCLOUD_BASE_URL=http://localhost/server \                                                                                                                                            
```

There seem to be some unexpected characters.

File: README.md [hunk 3]
```diff
@@ -574,6 +574,20 @@ cd $HOME/development/custom_apps/integration_openproject
 npm run watch
 ```
 
+### API tests
+The API tests start a full Nextcloud server and openProject_integration apps.
+To run the tests locally
+1. Setup nextcloud and enable `integration_openproject` apps.
+2. Goto path `server/apps/integration_openproject`
+2. Run `NEXTCLOUD_BASE_URL=http://localhost/server make api-test`
```
Comment (SagarGi) [inline README.md:582]:
```suggestion
3. Run `NEXTCLOUD_BASE_URL=http://localhost/server make api-test`
```

File: README.md [hunk 4]
```diff
@@ -574,6 +574,20 @@ cd $HOME/development/custom_apps/integration_openproject
 npm run watch
 ```
 
+### API tests
+The API tests start a full Nextcloud server and openProject_integration apps.
```
Comment (SagarGi) [inline README.md:578]: i cannot understand this?

File: README.md [hunk 5]
```diff
@@ -574,6 +574,20 @@ cd $HOME/development/custom_apps/integration_openproject
...
+To run the tests locally
+1. Setup nextcloud and enable `integration_openproject` apps.
+2. Goto path `server/apps/integration_openproject`
```
Comment (SagarGi) [inline README.md:581]: these are not needed.

**Review by SagarGi — CHANGES_REQUESTED** [review]
I would suggest to make 2 command only.
1. Running all the suites
2. Running a specific suites or scenario 
Just Keep it simple.

File: README.md [hunk 6]
```diff
@@ -573,6 +573,21 @@ Now you can watch for the app code changes using the following command and start
 cd $HOME/development/custom_apps/integration_openproject
 npm run watch
 ```
+> **_NOTE:_**  
+> Before API tests run, nextcloud instance needs to be ready, and also integration app needs to be enabled
```
Comment (phil-davis) [inline README.md:577]:
```suggestion
> Before running the API tests, the nextcloud instance needs to be ready, and also the integration app needs to be enabled
```

File: README.md [hunk 7]
```diff
@@ -573,6 +573,21 @@ Now you can watch for the app code changes using the following command and start
...
+
+### API tests
+To run the whole acceptance test locally run the command below.
```
Comment (phil-davis) [inline README.md:580]:
```suggestion
To run the whole of the acceptance tests locally run the command below.
```

**Review by SagarGi — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@nabim777 needs rebase to resolve the conflict for `makefile`

**Review by phil-davis — APPROVED** [review]

**Pull request merged**